spring-boot-autoconfigure/src/main/java/org/axonframework/springboot/updates/UpdateCheckConfiguration.java

```diff
@@ -0,0 +1,46 @@
+package org.axonframework.springboot.updates;
+
+import org.axonframework.updates.configuration.UsagePropertyProvider;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@ConfigurationProperties(prefix = "axon.update-check")
+public class UpdateCheckConfiguration implements UsagePropertyProvider {
+
+    /**
+     * Indicates whether the update check should be disabled. Defaults to detecting this setting via system properties or
+     * environment variables.
+     * <p>
+     * Unless disabled in any one of these locations, the update check will be enabled.
+     */
+    private Boolean disabled;
```

Comment:
Same thing here; unless we like dealing with `null` here (it is not initialized, so would be `null` if Spring never touches it I think).

```suggestion
    private boolean disabled;
```